src/lib/db/suggest-change-store.ts

```diff
@@ -249,6 +250,9 @@ export class SuggestChangeStore implements ISuggestChangeStore {
             { feature: change.feature, data: change.payload },
             user,
         );
+        if (shouldReturn) {
+            return this.get(changeSetID);
+        }
```

Comment:
Why do we need to make this conditional? Can we not just return the change set regardless and ignore it, if not?

Reply:
If it's for performance reasons, is there any way to encode that it will always be returned when you pass `shouldReturn`? Seems like a hassle to have to check whether a function returned `void` or an actual object 💁🏼

Reply:
I'm not sure I understand: you need the change set returned in both cases you mentioned over? Why would you then return `void`? Or maybe I misunderstood what you meant.

Reply:
In the `create` method this code adds the individual changes. I don't want to call get after each add - I want to call it when the whole `create` method is done 
```
suggestChangeSet.changes.forEach((change) => {
            this.addChangeToSet(change, id, user, false);
        });
```

Reply:
Ah, I think I see now; thanks for explaining 😄  Yeah, this doesn't sound like anything this function should care about to me. You already need the change set ID to add the change regardless, so if we think it would cause issues to always return the updated change set, then I'd personally vote to leave it up to the caller after updating.

Reply:
Agree with Thomas on this one

Reply:
Removed - moved the return to the service layer